index.html

```diff
@@ -9,7 +9,7 @@
     <link rel="stylesheet" href="header.css">
     <link rel="stylesheet" href="footer.css">
     <link rel="stylesheet" href="btn.css">
-
+    <link rel="stylesheet" href="cookie.css">
```

Comment:
There will be 5 HTTP requests to CSS files now, so please combine these to one single file before shipping. Added #27